src/builders/indexable-date-archive-builder.php

```diff
@@ -44,6 +57,8 @@ public function build( $indexable ) {
 		$indexable->is_robots_noindex = $this->options->get( 'noindex-archive-wpseo' );
 		$indexable->is_public         = ( (int) $indexable->is_robots_noindex !== 1 );
 		$indexable->blog_id           = \get_current_blog_id();
+		$indexable->permalink         = null;
+		$indexable->version           = $this->version;
```

Comment:
ah, well spotted. This is a remnant for when I wanted each builder to build a _complete_ object - but it seems unnecessary now :-( Let me think about this.

Reply:
we're keeping the lines in the builder; these are instantly clear to anyone who reads them. The line in the indexable_builder is a bit more 'magic' and we need less confusion, so that line was removed.